src/main/kotlin/app/revanced/patcher/util/ApkIdentifier.kt

```diff
@@ -0,0 +1,106 @@
+package app.revanced.patcher.util
+
+import com.reandroid.apk.ApkModule
+import com.reandroid.arsc.value.ResConfig.Density
+
+/**
+ * Utilities for determining the identity of an [ApkModule].
+ */
+object ApkIdentifier {
```

Comment:
This object class isn't really object oriented and should be merged with `Apk` class. The reason for that is, because it is only used to identify the type of the `Apk` which should happen when loading the class. A list of `Apk` files can be created when parsing them and a when statement can be used, to determine which type of `Apk` is present.